tests/test_pmft.py

```diff
@@ -182,158 +181,46 @@ def test_points_ne_query_points(self):
-        points = np.array([[-1.0, 0.0, 0.0], [1.0, 0.1, 0.0]],
-                          dtype=np.float32)
-        angles = np.array([0.0, np.pi/2], dtype=np.float32)
-        max_x = 3.6
-        max_y = 4.2
-        nbins_x = 20
-        nbins_y = 30
-        nbins_t = 40
-        dx = (2.0 * max_x / float(nbins_x))
-        dy = (2.0 * max_y / float(nbins_y))
-        dT = (2.0 * np.pi / float(nbins_t))
-
-        # calculation for array idxs
-        def get_bin(query_point, point, query_point_angle, point_angle):
-            r_ij = point - query_point
-            orientation = rowan.from_axis_angle([0, 0, 1], -point_angle)
-            rot_r_ij = rowan.rotate(orientation, r_ij)
-            xy_bins = np.floor((rot_r_ij[:2] + [max_x, max_y]) /
-                               [dx, dy]).astype(np.int32)
-            angle_bin = np.floor(
-                ((query_point_angle - np.arctan2(-r_ij[1], -r_ij[0])) %
-                 (2. * np.pi)) / dT).astype(np.int32)
-            return [xy_bins[0], xy_bins[1], angle_bin]
-
-        correct_bin_counts = np.zeros(shape=(nbins_x, nbins_y, nbins_t),
-                                      dtype=np.int32)
-        bins = get_bin(points[0], points[1], angles[0], angles[1])
-        correct_bin_counts[bins[0], bins[1], bins[2]] = 1
-        bins = get_bin(points[1], points[0], angles[1], angles[0])
-        correct_bin_counts[bins[0], bins[1], bins[2]] = 1
-        absoluteTolerance = 0.1
+    def get_bin(self, query_point, point, query_point_orientation,
+                point_orientation):
 
-        r_max = np.sqrt(max_x**2 + max_y**2)
-        test_set = util.make_raw_query_nlist_test_set(
-            box, points, points, 'ball', r_max, 0, True)
-        for nq, neighbors in test_set:
-            myPMFT = freud.pmft.PMFTXYT(max_x, max_y,
-                                        (nbins_x, nbins_y, nbins_t))
-            myPMFT.compute(nq, angles, neighbors=neighbors, reset=False)
-            npt.assert_allclose(myPMFT.bin_counts, correct_bin_counts,
-                                atol=absoluteTolerance)
-            myPMFT.compute(nq, angles, neighbors=neighbors)
-            npt.assert_allclose(myPMFT.bin_counts, correct_bin_counts,
-                                atol=absoluteTolerance)
+        r_ij = point - query_point
+        rot_r_ij = rowan.rotate(query_point_orientation, r_ij)
 
-            myPMFT.compute(nq, angles, neighbors=neighbors)
-            npt.assert_allclose(myPMFT.bin_counts, correct_bin_counts,
-                                atol=absoluteTolerance)
+        limits = np.asarray(self.limits)
+        xy_bins = tuple(np.floor((
+            rot_r_ij[:2] + limits) * np.asarray(self.bins[:2]) /
+            (2*limits)).astype(np.int32))
 
-    def test_repr(self):
-        max_x = 3.0
-        max_y = 4.0
-        nbins = 20
-        myPMFT = freud.pmft.PMFTXYT(max_x, max_y, nbins)
-        self.assertEqual(str(myPMFT), str(eval(repr(myPMFT))))
+        twopi = 2*np.pi
```

Comment:
Generally I recommend putting constants in all caps like `TWO_PI`. Perhaps even at the module level since it's reused.

Reply:
Done.